cfme/cloud/provider/__init__.py

```diff
@@ -272,6 +274,28 @@ def step(self):
         mon_btn('Timelines')
 
 
+@navigator.register(CloudProvider, 'Instances')
```

Comment:
@psav do we want these provider relationships as navmazing destinations?

Existing tests used Instance/Image by provider for navigation so I provided them, instead of going directly to the Instance/Image page (if applicable depending on CFME version).